README.md

```diff
@@ -128,9 +132,9 @@ Recomputes embeddings.
 | ------------------------- | ------ | --------- | ----------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
 | `path_to_known_faces`     | string | Optional  | `None`                              | Path to a file or database containing images or embeddings of known faces.                                       |
 | `face_detector_device`    | string | Optional  | `'cpu'`                             | Device on which the face detector will run. Options are `cpu` and `cuda`.                                         |
-| `face_detector_model`     | string | Optional  | `'ultraface_version-RFB-320-int8'` | Name of the model used for face detection.  Only option at the moment.                                                                        |
+| `face_detector_model`     | string | Optional  | `'ultraface_version-RFB-320-int8'` | Name of the model used for face detection. Only option at the moment.                                                                        |
 | `face_detection_threshold`| float  | Optional  | `0.9`                               | Confidence threshold for detecting faces, with values ranging from 0.0 to 1.0.                                   |
-| `face_feature_extractor_model` | string | Optional | `'facenet'` | Model used for extracting features from detected faces for identification.  Only option at the moment.                                      |
+| `face_feature_extractor_model` | string | Optional | `'facenet'` | Model used for extracting features from detected faces for identification. Only option at the moment.                                      |
 | `cosine_id_threshold`     | float  | Optional  | `0.3`                               | Threshold for determining face identity matches using cosine similarity.                                              |
 | `euclidean_id_threshold`  | float  | Optional  | `0.9`                               | Threshold for determining face identity matches using Euclidean distance.                                             |
 
```

Comment:
out of diff but on line 145: why no default for required and one for the optional?